docs/networking/kubeovn-vpc.md

```diff
@@ -0,0 +1,667 @@
+##  **Virtual Private Cloud (VPC) - Concepts & Architecture** 
```

Comment:
Use the same format as other sub-pages in the networking folder to make this page the entry point for KubeOVN.

In this case, this should be a page titled `Overlay VM Network` or something similar. 

<img width="283" height="255" alt="Image" src="https://github.com/user-attachments/assets/ad9dd8b3-3bce-4f54-afcb-d9bd1f031e35" />

cc @harvester/network